### PR TITLE
Add missing cluster metadata to multiple k8s metricsests

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## What does this PR do?

Add `orchestrator.cluster.name` and `orchestrator.cluster.ip` fields to the datasets: `proxy`, `system`, `apiserver`, `scheduler` and `controllermanager`.

## Why is it important?

`orchestrator.cluster.name` is used for kubernetes dashboards

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

- Related https://github.com/elastic/beats/issues/30425

## Screenshots
Screenshot 2022-09-05 at 16 47 45
Screenshot 2022-09-05 at 16 48 01

Screenshot 2022-09-05 at 16 46 50
Screenshot 2022-09-05 at 16 47 16
Screenshot 2022-09-05 at 16 47 31

### CODE REVIEW

File: metricbeat/module/kubernetes/util/kubernetes.go [hunk 1]
```diff
@@ -574,3 +574,24 @@ func GetClusterECSMeta(cfg *conf.C, client k8sclient.Interface, logger *logp.Log
 	}
 	return ecsClusterMeta, nil
 }
+
+// AddClusterECSMeta adds ECS orchestrator fields
+func AddClusterECSMeta(base mb.BaseMetricSet) mapstr.M {
```
Comment (ChrsMark) [inline metricbeat/module/kubernetes/util/kubernetes.go:579]: Can this be also used at https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/event/event.go#L102

**Review by ChrsMark — COMMENTED** [review]
lgtm, left one minor comment.

File: metricbeat/module/kubernetes/apiserver/apiserver.go [hunk 2]
```diff
@@ -22,48 +22,46 @@ import (
...
-			"apiserver_request_duration_seconds":      prometheus.Metric("request.duration.us", prometheus.OpMultiplyBuckets(1000000)),
-			"apiserver_request_total":                 prometheus.Metric("request.count"),
-			"apiserver_current_inflight_requests":     prometheus.Metric("request.current.count"),
-			"apiserver_longrunning_gauge":             prometheus.Metric("request.longrunning.count"),
-			"apiserver_storage_objects":               prometheus.Metric("etcd.object.count"),
-			"apiserver_audit_event_total":             prometheus.Metric("audit.event.count"),
-			"apiserver_audit_requests_rejected_total": prometheus.Metric("audit.rejected.count"),
-			"rest_client_requests_total":              prometheus.Metric("client.request.count"),
-		},
+var mapping = &prometheus.MetricsMapping{
```
Comment (gizas) [inline metricbeat/module/kubernetes/apiserver/apiserver.go:25]: As long as you correctly moved them out just you can consider use the bulk var definition as we have them here: https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/state_pod/state_pod.go#L36

Just to be the same.

**Review by gizas — APPROVED** [review]

**Pull request merged**